docs/tutorials/growth.md

```diff
@@ -34,4 +34,208 @@ featured: true
 
 Take advantage of yellow.ai's robust reporting capabilities to identify & drill down to the metrics that matter to you.
 
-Create custom reports & dashboards with data explorer for easy review and take control of your #data for actionable insights.
\ No newline at end of file
+Create custom reports & dashboards with data explorer for easy review and take control of your #data for actionable insights.
+
+
+## Data Explorer & Dashboards
+
+
+### Data Explorer
```

Comment:
This section needs to be migrated to platform concepts > growth

Reply:
According top Sanskity, below is the order she wanted under Insights:

API Usage
User Journeys
Data Explorer
Dashboards.

Reply:
If it's already published under platform concepts, remove it from the videos section. Only video content should be published here.

Reply:
Nothing was added or removed to/from the Videos section.